pkg/types.go

```diff
@@ -52,14 +52,34 @@ type CronWorkflow struct {
 }
 
 type WorkflowTemplate struct {
-	ID         uint64
-	CreatedAt  time.Time `db:"created_at"`
-	UID        string
-	Name       string
-	Manifest   string
-	Version    int32
-	IsLatest   bool `db:"is_latest"`
-	IsArchived bool `db:"is_archived"`
+	ID                               uint64
+	CreatedAt                        time.Time `db:"created_at"`
+	UID                              string
+	Name                             string
+	Manifest                         string
+	Version                          int32
+	IsLatest                         bool `db:"is_latest"`
+	IsArchived                       bool `db:"is_archived"`
+	WorkflowExecutionStatisticReport *WorkflowExecutionStatisticReport
+}
+
+type WorkflowExecutionStatisticReport struct {
+	Total        int32
+	LastExecuted time.Time
+	Running      int32
+	Completed    int32
+	Failed       int32
+}
+
+type WorkflowExecutionStatistic struct {
+	ID                 uint64
+	WorkflowTemplateId uint64
+	Name               string
+	Namespace          string
+	//Interface to support null values for timestamps, when scanning from db into structs
+	CreatedAt  interface{} `db:"created_at"`
+	FinishedAt interface{} `db:"finished_at"`
+	FailedAt   interface{} `db:"failed_at"`
```

Comment:
@aleksandrmelnikov instead of interface{}, would `*time.Time` work?

Reply:
No, because the db value can be null for finished_at or failed_at.
Squirrel throws an error because it can't assign "nil" to "time.Time"

Reply:
Actually, make sense. Pushed up change.